api_v3/services/MediaService.php

```diff
@@ -149,12 +149,20 @@ protected function replaceResource(KalturaResource $resource, entry $dbEntry, $c
 		}
 		else
 		{
-
 			$tempMediaEntry = new KalturaMediaEntry();
-		 	$tempMediaEntry->type = $dbEntry->getType();
+			$tempMediaEntry->type = $dbEntry->getType();
 			$tempMediaEntry->mediaType = $dbEntry->getMediaType();
-			$tempMediaEntry->conversionProfileId = $dbEntry->getConversionQuality();
 
+			if ( !$conversionProfileId ) {
+				$originalConversionProfileId = $dbEntry->getConversionQuality();
+				$conversionProfile = conversionProfile2Peer::retrieveByPK($originalConversionProfileId);
+				if ( is_null($conversionProfile) || $conversionProfile->getType() == ConversionProfileType::LIVE_STREAM )
+				{
+					$conversionProfileId = $this->getPartner()->getDefaultConversionProfileId();
```

Comment:
@tan-tan-kanarek please review fixes in this pull request: https://github.com/kaltura/server/pull/2418 
Thanks